.github/workflows/test.yml

```diff
@@ -15,7 +20,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          submodules: recursive
+          submodules: false
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: npm install --frozen-lockfile
```

Comment:
I saw that you have a `bun.lock` file on the root of the repo, which makes me think that you're using bun instead of npm. Is that right?

Reply:
True 🙆‍♂️